vite.config.ts

```diff
@@ -69,13 +81,19 @@ export default defineConfig(({ command, mode }) => {
                 'react-native-svg': 'react-native-svg-web',
                 'react-native-webview': 'react-native-web-webview',
                 'lottie-react-native': 'react-native-web-lottie',
+                '@react-native-clipboard/clipboard': 'react-native-web-clipboard',
                 jsbi: path.resolve(__dirname, '.', 'node_modules', 'jsbi', 'dist', 'jsbi-cjs.js'), // https://github.com/Uniswap/sdk-core/issues/20#issuecomment-1559863408
             },
             dedupe: ['react', 'ethers', 'react-dom', 'native-base'],
         },
         build: {
             commonjsOptions: {
+                extensions: ['.js', '.jsx', '.web.js', '.web.jsx'],
                 transformMixedEsModules: true, //handle deps that use "require" and "module.exports"
+                include: [/node_modules/],
+            },
+            rollupOptions: {
+                plugins: [jsxTransform([/react-native-.*\.jsx?$/])], //for some reason react-native packages are not being transpiled even with esbuild jsx settings
```

Comment:
what changed that this is needed?

Reply:
I had resolutions issues with one of the dependencies.

Retested now without this, and still works so not sure what changed but seems unnecessary to keep.
https://github.com/GoodDollar/GoodProtocolUI/pull/578/commits/692359963eda04c88b2f188165dbf8d82fa02128